<?xml version="1.0"?>
<launch>
    <node pkg="points_cluster" type="points_cluster_node" name="points_cluster_node" output="screen">
        <param name="sub_topic" value="/fusion_points_no_ground"/>
        <param name="pub_topic" value="/objects"/>

        <param name="show_objects_num" value="true"/>
        <param name="show_time" value="true"/>

        <param name="fit_obb" value="true"/>
        
        <param name="min_cluster_points_num" value="5"/>
        <param name="max_cluster_points_num" value="50000"/>
        
        <param name="min_cluster_size" value="0.3"/>
        <param name="max_cluster_size" value="3"/>
        
        <param name="seg_num" value="30"/>
        <rosparam param="seg_distance" > [30, 45, 60] </rosparam>
        <rosparam param="cluster_distance" > [0.5, 0.75, 1.0] </rosparam>

        <param name="road_info" value="true"/>
        <rosparam param="road_edge_left" > [0, 0, 0, 11.75] </rosparam>
        <rosparam param="road_edge_right" > [0, 0, 0, -8.25] </rosparam>
    </node>
</launch>
